src/kOS/Screen/TermWindow.cs

```diff
@@ -155,7 +161,7 @@ private void LoadFontArray()
                 // SetPixel on the texture (which we do).  So we start it off as a ARGB32
                 // first, long enough to perform the SetPixel call, then compress it
                 // afterward into a DXT5:
-                Texture2D charImage = new Texture2D(CHAR_SOURCE_SIZE, CHAR_SOURCE_SIZE, TextureFormat.ARGB32, true);
+                Texture2D charImage = new Texture2D(charSourceSize, charSourceSize, TextureFormat.ARGB32, false);
```

Comment:
The very last part of this one single line is actually the relevant bug fix - turning that `true` into a `false`.  Everything else in the PR is just the effect of making CHAR_SOURCE_SIZE no longer be hardcoded.